src/main/java/seedu/address/model/person/github/repo/RepoList.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.model.person.github.repo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Container for all the repositories of the class.
+ */
+public class RepoList {
+    private List<Repo> repoList = new ArrayList<>();
+
+    /**
+     * Constructor for RepoList Class.
+     */
+    public RepoList() {}
+
+    public List<String> getRepoNames() {
+        return repoList.stream().map(repo -> repo.getRepoName()).collect(Collectors.toList());
+    }
+
+    public int getNumberOfRepos() {
+        return repoList.size();
+    }
+}
```

Comment:
Lacking of `add`, `addAll`, `setAll` method to modify it. Also should it be a immutable list, so that when you call `add` method, it returns a new list.

Reply:
Have incorporated the former for modification.